xbmc.py

```diff
@@ -39,13 +38,12 @@
 TRAY_CLOSED_NO_MEDIA = 64
 TRAY_OPEN = 16
 
-
 # Custom AddonData type
 AddonData = namedtuple('AddonData', [
-    'kodi_home_path',       # data path (either portable of user path)
-    'add_on_id',            # the add-on id
-    'add_on_path',          # the full path to the add-on
-    'kodi_profile_path'     # the full path to the add-on profile folder
+    'kodi_home_path',  # data path (either portable of user path)
```

Comment:
Could you revert this change in comment alignment? Somehow my OCD-brain can't handle it :)

Reply:
haha, my pycharm tried 4 times to realign it :) Last time I gave in

Reply:
@basrieter updated